lib/live_cluster/client/node.py

```diff
@@ -165,58 +216,110 @@ async def __init__(
         self.sys_default_user_id = None
         self.sys_default_pwd = None
         self.sys_default_ssh_key = None
-        self.sys_cmds = [
-            # format: (command name as in parser, ignore error, command list)
-            ("hostname", False, ["hostname -I", "hostname"]),
-            ("top", False, ["top -n1 -b", "top -l 1"]),
-            (
+
+        _SysCmd.set_uid(os.getuid())
+        self.sys_cmds: list[_SysCmd] = [
+            # format: (command name as in parser, ignore error, sudo, command list)
```

Comment:
This format looks wrong. Isn't it command_name, ignore_error, command_list, parser?

Reply:
Forgot to delete the comment